core/new-gui/src/app/workspace/types/workflow-common.interface.ts

```diff
@@ -100,7 +100,13 @@ export type BreakpointTriggerInfo = Readonly<{
   operatorID: string;
 }>;
 
-export type PythonPrintTriggerInfo = Readonly<{
+export type PythonConsoleMessage = Readonly<{
```

Comment:
can we add the file path of the corresponding scala file

Reply:
will add in the next PR as it also updates this reference.